src/main/java/com/facebook/bitEncoderDecoder/bitwise/BitwiseEncoderImpl.java

```diff
@@ -8,4 +8,16 @@
     public String encode(String input) {
         return null;
     }
+
+
+    private String charHexToBin(Character ch) {
+        String bin = Integer.toBinaryString(ch);
+        StringBuilder result = new StringBuilder();
+        if (bin.length() == 6) {
```

Comment:
I do not understand this part. It seems that this code handles binary strings with length 6-8. What if bin has 5 length?